File: C/Factorial.c
#include <stdio.h>

int main() {
    int num, Cout;
    unsigned long long factorial = 1;

    
    printf("Input the number : ");
    scanf("%d", &num);

   
    if (num < 0)
        printf("Factorial of a negative number doesn't exist.\n");
    else {
        for(Cout = 1; Cout <= num; ++Cout) {
            factorial *= Cout;
        }
        printf("The Factorial of %d is: %llu\n", num, factorial);
    }

    return 0;}

File: Java/Factorial.java
public class Factorial {
    public static int factorial(int number) {
        if (number == 0 || number == 1) return 1;
        return n * factorial(number - 1);
    }

    public static void main(String[] args) {
        int num = 5;
        System.out.println("Factorial of " + num + " is " + factorial(num));
    }
}
